qiskit/visualization/latex.py

```diff
@@ -292,8 +292,8 @@ def _get_image_depth(self):
         for layer in self.ops:
             column_width = 1
             for nd in layer:
-                if nd.name in ['cu1', 'rzz']:
-                    column_width = 2
+                if nd.name in ['cu1', 'cp', 'rzz']:
+                    column_width = 3
```

Comment:
why?

Reply:
Previously we only showed the phase for 'u1', such as (pi/2). Now we add the name. Below is =3 on the left, =2 on the right.
![image](https://user-images.githubusercontent.com/16268251/108859338-f8d7aa80-75a1-11eb-8f9c-79bb85e6ade1.png)